web/src/pwa.ts

```diff
@@ -0,0 +1,20 @@
+import { registerSW } from 'virtual:pwa-register';
```

Comment:
there is also a specific react one that comes as a hook. This might work better for only including in the stat keeper